src/js-form-validator.ts

```diff
@@ -1,4 +1,11 @@
-import { closest } from './github/GTMSportswear/js-utilities@1.0.0/js-utilities';
+import { closest } from './github/gtmsportswear/js-utilities@1.0.0/js-utilities';
+
+enum ValidationTypes {
```

Comment:
It seems to me that this should be singular, so `ValidationType`. Also, could it be `ValidationStatus` since it is more of a status than a type?